src/alchemlyb/parsing/gmx.py

```diff
@@ -29,7 +29,9 @@ def extract_u_nk(xvg, T):
 
     """
 
-    col_match = r"\xD\f{}H \xl\f{}"
+    h_col_match = r"\xD\f{}H \xl\f{}"
+    pv_col_match = 'pV'
+    u_col_match = 'Total Energy'
```

Comment:
Checking the files you provided @harlor (https://userpage.fu-berlin.de/dominikwille/nvt300K.tgz), I see "Potential Energy (kJ/mol)" in the `ti_0.xvg` file. Should we also try and match that?

Reply:
@mrshirts explained that in https://github.com/alchemistry/alchemlyb/issues/59#issuecomment-430721922

I uploaded a dataset with total energy here: http://dominikwille.userpage.fu-berlin.de/nvt300K_u.tgz

Reply:
Okay, but I *think* [depending on MDP options given to gromacs](https://alchemlyb.readthedocs.io/en/latest/parsing/alchemlyb.parsing.gmx.html) it's possible to have a dataset without "Total Energy" but with "Potential Energy". I was thinking that we could handle this in [another PR with kwargs](https://github.com/alchemistry/alchemlyb/pull/62#issuecomment-434935124), but I don't believe this should be neglected.

Could you add a check for the existence of a "Potential Energy" column as you did for the "Total Energy" column, and if present, use this instead of the "Total Energy" column? That would satisfy me for the purposes of this PR